snekbox/nsjail.py

```diff
@@ -44,19 +44,15 @@ class NsJail:
     def __init__(self, nsjail_binary: str = NSJAIL_PATH):
         self.nsjail_binary = nsjail_binary
         self.config = self._read_config()
+        self.cgroups_version = self._probe_cgroup_version()
 
-        log.info(f"Cgroups version: {self._probe_cgroup_version()}")
+        log.info(f"Cgroups version: {self.cgroups_version}")
 
     @staticmethod
     def _probe_cgroup_version() -> int:
         """Poll the filesystem and return the guessed cgroup version."""
         # Right now we check whenever the controller path exists
-        version = 2 if CGROUPV2_PROBE_PATH.exists() else 1
-
-        if DEBUG:
-            log.info(f"Guessed cgroups version: {version}")
-
-        return version
+        return 2 if CGROUPV2_PROBE_PATH.exists() else 1
```

Comment:
This doesn't even need to be a function. Just do `self.cgroups_version = 2 if CGROUPV2_PROBE_PATH.exists() else 1`.